CHANGELOG.md

```diff
@@ -1,132 +1,136 @@
+# 8.8.0 02-Apr-2020
+
+- Add support for Item add token endpoint (BETA) ([#269](https://github.com/plaid/plaid-ruby/pull/269))
+  - `/item/add_token/create`
+
 # 8.7.0 05-Mar-2020
 
-* Add transaction_code to Transaction class ([#264](https://github.com/plaid/plaid-ruby/pull/264))
-* Add OAuth support to /institutions endpoints ([#261](https://github.com/plaid/plaid-ruby/pull/261))
+- Add transaction_code to Transaction class ([#264](https://github.com/plaid/plaid-ruby/pull/264))
+- Add OAuth support to /institutions endpoints ([#261](https://github.com/plaid/plaid-ruby/pull/261))
```

Comment:
while `*` is technically valid markdown in some flavors, its often confusing when trying to use bold or italics as they use the same markers and can end up matching the "bullet point" at the beginning of the line. `-` is not used for anything else and seems to be standardized by Prettier and other markdown editors.

Reply:
👍